dll/win32/browseui/shellfind/CFindFolder.cpp

```diff
@@ -671,11 +671,17 @@ STDMETHODIMP CFindFolder::GetDetailsOf(PCUITEMID_CHILD pidl, UINT iColumn, SHELL
     if (!pidl)
         return SHSetStrRet(&pDetails->str, _AtlBaseModule.GetResourceInstance(), g_ColumnDefs[iColumn].iResource);
 
-    if (iColumn == 1)
+    if (iColumn == IDS_COL_LOCATION_COLNUM)
     {
         return SHSetStrRet(&pDetails->str, _ILGetPath(pidl));
     }
 
+    if (iColumn == IDS_COL_RELEVANCE_COLNUM)
+    {
+        /* TODO: Fill once the relevance is calculated */
```

Comment:
this is a cpp file. You should not hesitate the slightest bit to favor CPP // commenting style over C /* */

Reply:
Done